broker/src/test/java/io/zeebe/broker/system/configuration/BrokerCfgTest.java

```diff
@@ -836,13 +847,13 @@ private void assertEmbeddedGatewayEnabled(final String configFileName, final boo
     assertThat(gatewayCfg.isEnable()).isEqualTo(enabled);
   }
 
-  private void assertDefaultDebugLogExporter(final boolean prettyPrint) {
-    assertDebugLogExporter("default", prettyPrint);
-    assertDebugLogExporter("empty", prettyPrint);
+  private void assertDefaultDebugLogExporter() {
```

Comment:
```suggestion
  private void assertDebugLogExporterRegistered() {
```

The name is confusing. In line 262 ` public void shouldHaveNoExportersByDefault() {` we test for having no exporter by default.

Here, with the name `Default` in it we would be testing that there is an exporter registered. But the only place we call it from is not a vanilla default configuration.